client/pd_service_discovery.go

```diff
@@ -239,17 +241,19 @@ func (c *pdServiceDiscovery) updateMemberLoop() {
 	ticker := time.NewTicker(memberUpdateInterval)
 	defer ticker.Stop()
 
+	bo := retry.InitialBackOffer(updateMemberBackOffBaseTime, updateMemberTimeout)
```

Comment:
@lhy1024 for `member loop` max backup time time is `1 second` which is `updateMemberTimeout`, PTAL, thx!

Reply:
Got it.